lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'menu'
+
+class Order
+
+attr_reader :user_input, :current_order, :meal_quantities, :meal_selection
+
+  def initialize(user_input, menu_choice)
+    @user_input = user_input
+    @current_order = current_order
+    @menu_choice = menu_choice
+    @bill = 0
+  end
+
+
+
+  def process_order
+    @meal_quantities = user_input.gsub(/^0-9/, " ").split(" ").map!{|number| number.to_i}
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SymbolProc: Pass `&:to_i` as an argument to `map!` instead of a block.